fixtures/generic/color-temperature-fader.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Color Temperature Fader",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Ken"],
+    "createDate": "2024-05-04",
+    "lastModifyDate": "2024-05-04"
+  },
+  "comment": "Generic fader + color temperature",
+  "links": {
+    "other": [
+      "http://example.com"
+    ]
+  },
+  "availableChannels": {
+    "Intensity": {
+      "fineChannelAliases": ["Intensity fine"],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Color Temperature": {
+      "fineChannelAliases": ["Color Temperature fine"],
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "warm",
+        "colorTemperatureEnd": "cold"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "8 bit",
+      "shortName": "8bit",
+      "channels": [
+        "Intensity",
+        "Color Temperature"
+      ]
+    },
+    {
+      "name": "16 bit",
+      "shortName": "16bit",
```

Comment:
I don't know that there's any 16-bit fixtures like this but it costs almost nothing to add.